Rock.Messaging/Configuration/XmlSerializingRockMessagingConfiguration.cs

```diff
@@ -2,7 +2,11 @@
 using System.Linq;
 using System.Xml.Serialization;
 
+#if ROCKLIB
+namespace RockLib.Messaging
```

Comment:
This class probably doesn't need to exist in the RockLib project (and if it doesn't, then this file doesn't need the precompiler directive).